user_manual/changes/UC08-034.yaml

```diff
@@ -0,0 +1,6 @@
+type: new-feature
+title: Make subtype_mark optional in object renames
+description: |
+    The subtype_mark in object renaming declarations is made optional as per
+    AI12-0275 (Ada 2022 feature).
```

Comment:
This is not sufficient: You're introducing a new node type in LAL, this needs to be documented here as well, since it's not merely as simple as making a field optional.